app/schema/portal/cdo/dog-details.js

```diff
@@ -1,5 +1,25 @@
 const Joi = require('joi')
-const { isFuture } = require('date-fns')
+const { UTCDate } = require('@date-fns/utc')
+const { isMatch, isValid, isFuture, addMonths } = require('date-fns')
+const { dateComponentsToString } = require('../../../lib/date-helpers')
+
+const validateDate = (value, helpers) => {
+  const format = isMatch(value, 'yyyy-MM-dd') ||
+    isMatch(value, 'yyyy-M-d', { locale: 'enGB' }) ||
+    null
+
+  if (!format) {
+    return helpers.message('CDO issue date must be a valid date')
+  }
+
+  const date = new Date(value)
```

Comment:
Is it possible that 'new Date(value)' may accept invalid dates but shift the days, for example, 30/02/2000 ? Can we cover that scenario in the unit tests?